### ISSUE TITLE
how to get the form to work

### ISSUE THREAD

**Issue by andreadesiderio** [digitalcraftsman/hugo-agency-theme#124]
I've tried simply adding my email in config like the instructions said, 
I've tried with netlify  = true in config and without it as well. 
I tried using my own captcha, I have also commented out the captcha section in contact.html

….. but my form will send no matter what I try.

I've been trying to get this to work for a week now and still can't find what I am doing wrong?

**Comment by stp-ip** [comment]
So with netlify = false, no postURL and email set it should use formspree, which might need a payed account depending on configuration. They changed this more recently.

The other option is to use netlify.com to host your site. When setting netlify = true this should work out of the box and appear within the netlify.com interface as a submission.

Also sidenote. This theme might not be as actively maintained as the currently developed [Syna Theme](https://syna.okkur.io)

**Comment by tomi96** [comment]
Thank you @stp-ip . 

When the agency.js is relabelled so its not used (relabelled to _agency.js) and  [params.contact.captcha] and sitekey is commented out formspree works. 
If the above  not done then "Message could not be send. 
Please contact us at "email address" instead." is shown. 
Is there something in agency.js we should be aware of?

**Comment by stp-ip** [comment]
captcha isn't supported with the agency's formspree I think. Also not sure, what breaks within agency.js. Might have been a recent change to formspree.

If you have more details we might be able to look, if this is a bug. Currently we only try to do bug fixes here and iterate on [Syna](syna.okkur.io), which is close to its first announced release (v0.15.0).

**Comment by NickoLabs** [comment]
So the only way I was able to get the form to work:
- Use Netlify;
- remove all captcha verification;
- remove the "toggleDisabled" module from the Form Validator;
- entirely disable the Ajax submission of the form;

That way, the form submission goes back to being default behaviour which is supported by Netlify.

My demo isn't going to be widely used so I don't mind not having any captcha, though I could probably resort to using a honey-pot method instead. However I wouldn't recommend a "free" form like that on a live website, so a real solution still is needed.

If anyone wanna take a look, I dropped it there https://github.com/NickoLabs/hugolabs 

Probably not the cleanest repository, since I've not been using GitHub a lot and pretty new to having to commit stuff over and over (and my mix of French and English in my code 😆 ), in a world where saving my file usually puts it on the FTP, but feel free to let me know if anything is totally wrong...
